tests/integration/test_all_sets_image_diffs.py

```diff
@@ -174,15 +69,16 @@ def test_area_mean_time_series_plot_diffs(self):
 
             # Check PNG path is the same as the expected.
             png_path = "{}/{}.png".format(set_name, variable)
-            full_png_path = "{}{}".format(self.results_dir, png_path)
+            full_png_path = os.path.join(self.results_dir, png_path)
```

Comment:
I fixed all of these paths by using `os.path.join`. Otherwise, a missing "/" would break these tests.